Portability fixes for musl taken from voidlinux:

- _WORDIZE is declared in <bits/user.h> via <sys/user.h>.

	--xtraeme

[Bernd: Downloaded from
http://git.alpinelinux.org/cgit/aports/tree/testing/pulseaudio/musl-fixes.patch
and adjusted paths.]
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
[Thomas: remove xlocale.h part, it was fixed upstream.]
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

--- a/configure.ac.orig	2015-09-24 11:36:06.107661526 +0200
+++ b/configure.ac	2015-09-24 11:36:40.420036775 +0200
@@ -407,7 +407,7 @@ AC_HEADER_STDC
 # POSIX
 AC_CHECK_HEADERS_ONCE([arpa/inet.h glob.h grp.h netdb.h netinet/in.h \
     netinet/in_systm.h netinet/tcp.h poll.h pwd.h sched.h \
-    sys/mman.h sys/select.h sys/socket.h sys/wait.h \
+    sys/mman.h sys/select.h sys/socket.h sys/wait.h sys/user.h \
     sys/uio.h syslog.h sys/dl.h dlfcn.h linux/sockios.h])
 AC_CHECK_HEADERS([netinet/ip.h], [], [],
                  [#include <sys/types.h>
--- a/src/pulsecore/sample-util.h.orig	2015-09-24 11:37:57.773677238 +0200
+++ b/src/pulsecore/sample-util.h	2015-09-24 11:38:46.668208677 +0200
@@ -24,6 +24,14 @@
 #include <inttypes.h>
 #include <limits.h>
 
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#ifdef HAVE_SYS_USER_H
+#include <sys/user.h>	/* required by _WORDIZE with musl */
+#endif
+
 #include <pulse/gccmacro.h>
 #include <pulse/sample.h>
 #include <pulse/volume.h>
